deploy/aks/monitoring/perf/README.md

```diff
@@ -0,0 +1,109 @@
+## Performance Test Dashboard Setup
+
+This document serves as a guideline for setting up a performance test dashboard using Azure managed Prometheus and Grafana. We will be creating an AKS (Azure Kubernetes Service) cluster and installing Prometheus Pushgateway and Azure Monitoring Agent (AMA) to scrape performance test metrics and push them to Azure managed Prometheus. Additionally, we will set up an ingress controller with authentication to act as a proxy for Prometheus Pushgateway. Finally, we will create a Grafana dashboard by importing a predefined JSON model.
+
+Below are step-by-step instructions to set up a performance test dashboard using Azure managed Prometheus and Grafana. Follow these steps to configure your performance monitoring environment.
+
+#### Step 1: Clone the test-infra repo
+
+```bash
+git clone https://github.com/dapr/test-infra.git
+cd deploy/aks/monitoring/perf
+```
+
+#### Step 2: Login to Azure
+
+```bash
+az login
```

Comment:
Something that I would add (because I have been bitten by this a number of times) is a command to ensure you are operating at the desired subscription. Performing log in does not lock commands to a particular subscription. This is less of an issue if you are logging in to an account that only has a single subscription but it becomes troublesome if you account (like ours) has access to multiple directories (logging in does not lock you to a directory) and if that directory itself hosts multiple subscriptions.

Something like the following would do the trick. I will send a PR changing the main README in this repo to do something to that effect

```bash
export SUBSCRIPTION_ID=<SUBSCRIPTION UUID TO BE USED FOR THE COMMANDS BELLOW>

# ...

az account set --subscription "${SUBSCRIPTION_ID}"
```